easybuild/easyconfigs/b/biogeme/biogeme-3.2.8-foss-2021a-Python-3.9.5.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'PythonBundle'
+
+name = 'biogeme'
+version = '3.2.8'
+versionsuffix = '-Python-%(pyver)s'
```

Comment:
Same here - please remove the `versionsuffix`.

Reply:
@verdurin removed!